content/en/blog/_posts/2021-10-05-nsa-cisa-hardening.md

```diff
@@ -402,11 +402,14 @@ In summary, it is fantastic to see security practitioners sharing this
 level of detailed guidance in public. This guidance further highlights
 Kubernetes going mainstream and how securing Kubernetes clusters and the
 application containers running on Kubernetes continues to need attention and focus of
-practitioners. Only a few weeks after the guidance was published, an open source
-tool [kubescape](https://github.com/armosec/kubescape) to validate cluster
-against this guidance became available.
+practitioners.
 
-This tool can be a great starting point to check the current state of your
```

Comment:
With newer tools like [cloudquery](https://github.com/cloudquery/cloudquery) being introduced in the open source landscape for validation and automation of this guidance, they can be a great starting point to check the current state of your clusters. Post which you can use the information in this blog post and in the guidance to assess where improvements can be made.